### DISCUSSION TITLE
Busco equipo para el Hackaton

### DISCUSSION THREAD

**Discussion by Fcovaldz** [Launch-X-Latam/MisionFrontEnd#587] [Team Posts]
Hola explorers buenas!!, busco equipo para el Hackaton mi correo es: lx00111@innovaccion.mx

**Comment by RobertRamos93** [comment]
Hola compañero, yo también busco equipo, me interesa unirme.

**Comment by ChristianTongCruz** [comment]
Le escribí al correo para poder coordinar, yo también quiero estar en el equipo para el Hackaton

**Comment by gapman20** [comment]
Hola tambien busco equipo para el hackaton aqui dejo mi correo por si aun tienes cupo lx16569@innovaccion.mx

**Comment by Fcovaldz** [comment]
Quien puede hacer el grupo donde estemos todos?

**Comment by StephanyCS1** [comment]
Yo también estoy buscando equipo mi correo es: lx09176@innovaccion.mx

**Comment by Kuritoo** [comment]
Hola, también estoy buscando equipo. Mi correo es: lx09026@innovaccion.mx

**Comment by shelkanife** [comment]
Yo igual busco equipo, mi correo es: lx15879@innovaccion.mx

**Comment by SNOOPYFC** [comment]
ya tienen equipo?

**Comment by kerizrh** [comment]
Hola, yo no tengo equipo ya llevo rato preguntando y nadie me responde para poder integrarme, agregren porfa y hablamos por Teams o por mails  Mi correo: lx07570@innovaccion.mx

**Comment by edgardorojas** [comment]
También estoy en busca de equipo, mi correo lx17007@innovaccion.mx. Saludos, felices fiestas

**Comment by kerizrh** [comment]
Hola yo también necesito un equipo.mi correo:  lx07570@innovaccion.mx

**Comment by RicardoLunaRivera** [comment]
aun hay cupo?
Dejo mi correo: lx04128@innovaccion.mx
